package.json

```diff
@@ -37,58 +70,30 @@
     "stylelint-config-standard": "^17.0.0",
     "supertest": "^3.0.0",
     "testpilot-ga": "^0.3.0",
-    "uglifyify": "^4.0.3"
+    "uglifyify": "^4.0.3",
+    "webpack": "^3.4.1"
   },
   "engines": {
     "node": ">=8.0.0"
   },
   "homepage": "https://github.com/mozilla/send/",
   "license": "MPL-2.0",
   "repository": "mozilla/send",
-  "availableLanguages": [
-    "en-US",
-    "zh-TW",
-    "zh-CN",
-    "cs",
-    "fr",
-    "fy-NL",
-    "de",
-    "hu",
-    "it",
-    "ja",
-    "kab",
-    "ms",
-    "nb-NO",
-    "nn-NO",
-    "pt-PT",
-    "pt-BR",
-    "ru",
-    "sk",
-    "sl",
-    "dsb",
-    "hsb",
-    "es-CL",
-    "es-ES",
-    "es-MX",
-    "sv-SE",
-    "tr",
-    "cy"
-  ],
   "scripts": {
     "build": "npm-run-all build:*",
-    "build:upload": "browserify frontend/src/upload.js -g uglifyify -o public/upload.js",
-    "build:download": "browserify frontend/src/download.js -g uglifyify -o public/download.js",
-    "build:version": "node scripts/version",
     "build:l10n": "cp node_modules/l20n/dist/web/l20n.min.js public",
+    "build:public": "npm run webpack -- # -p",
+    "build:version": "node scripts/version",
     "dev": "npm run build && npm start",
     "format": "prettier '{frontend/src/,scripts/,server/,test/**/!(bundle)}*.js' 'public/*.css' --single-quote --write",
     "lint": "npm-run-all lint:*",
     "lint:css": "stylelint 'public/*.css'",
     "lint:js": "eslint .",
     "start": "node server/server",
     "test": "npm-run-all test:*",
-    "test:unit": "mocha test/unit",
+    "test:browser": "browserify test/frontend/frontend.bundle.js -o test/frontend/bundle.js -d && node test/frontend/driver.js",
```

Comment:
This would need to change from **browserify** to **webpack** as well. The naming feels a bit funny, with renaming "frontend.bundle.js" to "bundle.js". As best I can guess, we'd create a special webpack.test.config.js or something and pass that as a custom config so we could have custom `output.name` and `output.path` properties.